src/test/java/org/javarosa/core/model/ChoiceNameTest.java

```diff
@@ -117,4 +119,75 @@ public class ChoiceNameTest {
         scenario.answer("/data/city", "grenoble");
         assertThat(scenario.answerOf("/data/city_name"), is(stringAnswer("Grenoble")));
     }
+
+    @Test public void choiceNameCallWithIndexedRepeatAndStaticChoices_worksWithMultipleRepeats() throws IOException, XFormParser.ParseException {
```

Comment:
This is not affected by the code change. It's there to document behavior that has always worked.